tools/prepare_python_system_tests.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import json
+import os
+import subprocess
+import tempfile
+import time
+import urllib.request
+import zipfile
+
+parser = argparse.ArgumentParser(description='Downloads the latest release artifacts from nimi-python and runs system tests on the specified python package.',
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-d', '--driver', required=True, type=str,
+                    help='Python package name.',
+                    choices=os.listdir('src'))
+parser.add_argument('-pv', '--python-version', required=False, type=str,
+                    help='Python version to be run. This is used to invoke the appropriate tox environment.',
+                    choices=['py35', 'py36', 'py37', 'py38', ], default='py38')
+parser.add_argument('-pb', '--python-bitness', required=False, type=str,
+                    help='Python bitness to be run. This is passed to tox.',
+                    choices=[None, '--32'], default=None)
```

Comment:
Seems weird that you cannot choose 64 bits. And it isn't clear what `None` does.
Consider making choices ["32", "64", "any"]. It will require the choices to do a lookup so you can pull out the string to pass `tox`.

Reply:
Updated.